src/Npgsql/Replication/Logical/NpgsqlLogicalReplicationConnection.cs

```diff
@@ -0,0 +1,12 @@
+﻿using JetBrains.Annotations;
+
+namespace Npgsql.Replication.Logical
+{
+    /// <summary>
+    /// Represents a logical replication connection to a PostgreSQL server
+    /// </summary>
+    public sealed class NpgsqlLogicalReplicationConnection : NpgsqlReplicationConnection
```

Comment:
We need a constructor here that accepts the connection string (same for physical connection).

Reply:
Done in f53b5818b721af74a8a3037be92efcc46733b7c6